pages/establishment/pdf/index.jsx

```diff
@@ -43,39 +44,16 @@ module.exports = settings => {
       })
       .then(() => {
         const store = createStore(stateReducer, initialState);
-        const html = renderToStaticMarkup(<Body establishment={establishment} nonce={res.locals.static.nonce} content={content} />);
+        const body = renderToStaticMarkup(<Body establishment={establishment} nonce={res.locals.static.nonce} content={content} />);
         const header = renderToStaticMarkup(<Header store={store} model={establishment} licenceType="pel" nonce={res.locals.static.nonce} />);
         const footer = renderToStaticMarkup(<Footer />);
 
         const hasStatusBanner = establishment.status !== 'active';
 
-        const params = {
-          method: 'POST',
-          json: {
-            template: html,
-            pdfOptions: {
-              displayHeaderFooter: true,
-              headerTemplate: header,
-              footerTemplate: footer,
-              margin: {
-                top: hasStatusBanner ? 180 : 100,
-                left: 25,
-                right: 25,
-                bottom: 125
-              }
-            }
-          }
-        };
-
-        fetch(`${settings.pdfService}/convert`, params)
-          .response
+        pdf({ body, header, footer, hasStatusBanner })
           .then(response => {
-            if (response.status < 300) {
-              res.attachment(`${establishment.name}.pdf`);
-              response.body.pipe(res);
-            } else {
-              throw new Error(`Error generating PDF - generator responded ${response.status}`);
-            }
+            res.attachment(`${establishment.name}.pdf`);
```

Comment:
we probably want to run the establishment name through filenamify as well just incase.

Reply:
That'll teach me to make whitespace changes...